datasetinsights/estimators/faster_rcnn.py

```diff
@@ -80,59 +141,60 @@ def __init__(
             box_score_thresh=box_score_thresh,
         )
         self.model_without_ddp = self.model
-        self.gpu = gpu
-        self.rank = rank
         self.sync_metrics = config.get("synchronize_metrics", True)
-        logger.info(f"gpu: {self.gpu}, rank: {self.rank}")
-
-        self.checkpointer = checkpointer
-        checkpoint_file = config.get("checkpoint_file", const.NULL_STRING)
-        if checkpoint_file != const.NULL_STRING:
-            checkpointer.load(self, config.checkpoint_file)
-
+        self.data_root = data_root
         self.metrics = {}
         for metric_key, metric in config.metrics.items():
             self.metrics[metric_key] = EvaluationMetric.create(
                 metric.name, **metric.args
             )
         self.model.to(self.device)
 
-        if self.config.system.distributed:
+        if self.distributed:
             self.model = torch.nn.parallel.DistributedDataParallel(
                 self.model, device_ids=[self.gpu]
             )
             self.model_without_ddp = self.model.module
 
+        self.checkpoint_file = self.config.get(
```

Comment:
[mustfix] checkpoint_file should be removed from config.

Reply:
Will do